nodemon.json

```diff
@@ -0,0 +1,37 @@
+{
+  "verbose": true,
+  "watch": [
+    "node_modules/oae-activity",
```

Comment:
Can we try using a regex (something like oae-*) instead? Otherwise we will need to update this file every time we change the modules (for example, the Jitsi PR adds a module and the Zencoder PR adds one and removes one).